src/esn.calendar.libs/app/components/event-date-edition/event-date-edition.controller.js

```diff
@@ -166,4 +183,9 @@ function calEventDateEditionController(esnI18nDateFormatService, esnDatetimeServ
   function _updateMinEndDate() {
     self.minEndDate = getMinEndDate();
   }
+
+  function _refreshTimeInputs() {
+    self.startTime = self.start.toDate();
+    self.endTime = self.end.toDate();
+  }
```

Comment:
Why this logic?

Reply:
check https://github.com/OpenPaaS-Suite/esn-frontend-calendar/pull/201/files#r502385815